CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 ## [unreleased] - aaaa-mm-dd
 
 ### Added
-*
+* ClientNetworkAnimator component has been added to the Samples Utilities Package. This allows for authority on Animators to be passed onto clients, meaning animations will be client-driven.
```

Comment:
There's a changelog file directly in the utilities package, would you mind moving this there?